pyproject.toml

```diff
@@ -0,0 +1,79 @@
+# ********************************
+# |docname| - Poetry configuration
+# ********************************
+#
+# See https://python-poetry.org/docs/dependency-specification/ to get an understanding of
+# how poetry does specifies dependencies.
+#
+# Project metadata
+# ================
+[tool.poetry]
+name = "runestone_web2py_server"
+version = "0.1.0"
+description = "A web2py-based server for the Runestone e-book system."
+authors = ["Your Name <you@example.com>"]
+license = "MIT"
+
+
+# Dependencies
+# ============
+[tool.poetry.dependencies]
+cryptography = "^3.0.0"
+altair = "^4.0.0"
+bookserver = "^0.3.0"
+bleach = "> 3.1.1"
+celery = { extras = ["redis,gevent"], version = "^5.0.0"}
+cssselect ">= 1.0"
```

Comment:
To answer my own question per (https://stackoverflow.com/questions/54720072/dependency-version-syntax-for-python-poetry) 

the ^ indicates to update to the newest version matching the leftmost non-zero digit.  which makes it easy to keep major versions the same.

Reply:
Thanks, fixed. I copied the `>=` from the existing `requirements.txt` if present, and otherwise use `^`. Do you want to move to using `^` for everything?

Reply:
They do have slightly different meanings as I noted above.  I think the ^ notation is nice because it prevents auto upgrades over a major release.    But we also have to be careful because book server is specified "^0.3.0" which means that when I release 0.4.0 it would be ignored. Probably I should make a release of it and call it 1.0.0 then we would only have to change the pyproject.toml file when we get to 2.0

Reply:
I'd suggest we get everything working, then switch to ^ instead of >= for the depenencies.